lonestar/analytics/distributed/README.md

```diff
@@ -23,7 +23,12 @@ For distributed Galois, i.e. D-Galois:
 
 For distributed and heterogeneous Galois, i.e. D-IrGL:
 
-`cmake ${GALOIS_ROOT} -DGALOIS_ENABLE_DIST=1 -DGALOIS_ENABLE_GPU=1`
+`cmake ${GALOIS_ROOT} -DGALOIS_ENABLE_DIST=1 -DGALOIS_CUDA_CAPABILITY=<insert CUDA capability here>`
+
+The CUDA capability should be one that your GPU supports. For example, if I
```

Comment:
I don't know what the doc style is for voice, but `I` here seems out of place to me, since that presumably refers to the author not the reader. "You" instead?

Reply:
"You" does make more sense.